<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>657</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #222;
}

/* 버튼 기본 상태 */
QPushButton {
    background-color: #444;
    color: white;
    border-radius: 5px;
    font-size: 18px;
    padding: 10px;
    border: none;
}

/* 버튼 마우스 오버 */
QPushButton:hover {
    background-color: #555;
}

/* 버튼 누를 때 */
QPushButton:pressed {
    background-color: #666;
}

/* 입력창 (Ans, 계산식 표시 라인) */
QLineEdit {
    background-color: #333;
    color: white;
    border: none;
    font-size: 20px;
    padding: 5px;
}
</string>
  </property>
  <widget class="QPushButton" name="pushButton_mod">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 20pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sign">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 20pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>±</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_all_clear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 20pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 20pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 20pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 20pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_div">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 20pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mul">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>380</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 20pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans Mono</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 500 20pt &quot;Ubuntu Sans Mono&quot;;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_plus">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>560</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 20pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans Mono</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 500 20pt &quot;Ubuntu Sans Mono&quot;;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sub">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>470</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 20pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 20pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 20pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>470</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans Mono</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 500 20pt &quot;Ubuntu Sans Mono&quot;;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>470</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 20pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_equal">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>560</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans Mono</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 500 20pt &quot;Ubuntu Sans Mono&quot;;</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dot">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>560</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 20pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_0">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 20pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>15</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 15pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>⌫</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_curr">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>30</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 30pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prev">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>51</green>
         <blue>51</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 20pt &quot;Ubuntu Sans&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
